### PR TITLE
Initial commit with flatpack manifest and appdata of Nootka application

### PR DESCRIPTION
**Pull request by SeeLook**
Please, add this app to flatpack repo

### CODE REVIEW

**Comment by nedrichards** [comment]
bot, build net.sf.nootka

File: nootka.appdata.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<application>
+  <id type="desktop">nootka.desktop</id>
+  <name>Nootka</name>
+  <metadata_license>CC0-1.0</metadata_license>
+  <summary>to play scores</summary>
```
Comment (nedrichards) [inline nootka.appdata.xml]: This isn't a great description.

File: nootka.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    </p>
+    <p>Features:</p>
+    <ul>
+      <li>interactive interface to discover the rules of musical notation</li>
+      <li>exercises with possibility to create own sets</li>
+    </ul>
+  </description>
+
+  <categories><category>Education</category><category>Music</category></categories>
+
```
Comment (nedrichards) [inline nootka.appdata.xml]: Needs version tags

**Comment by nedrichards** [comment]
Are you sending the appdata upstream? This is an actively developed application and it would benefit to have that upstream.

File: net.sf.nootka.json [hunk 3]
```diff
@@ -0,0 +1,123 @@
+{
+  "app-id": "net.sf.nootka",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.9",
+  "sdk": "org.kde.Sdk",
+  "command": "nootka",
+  "branch": "1.4.5",
```
Comment (nedrichards) [inline net.sf.nootka.json]: on flathub everything has to be branch: stable - just remove this bit.

File: net.sf.nootka.json [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+  "rename-desktop-file": "nootka.desktop",
+  "rename-icon": "nootka",
+
+  "finish-args":[
+      "--device=dri",
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--share=network",
+      "--filesystem=host",
```
Comment (nedrichards) [inline net.sf.nootka.json]: This is an excessive permission, what do you need all of host for?

File: net.sf.nootka.json [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+        {
+          "type": "archive",
+          "url": "https://sourceforge.net/projects/nootka/files/Nootka-1.4.5/nootka-1.4.5-source.tar.bz2",
+          "sha256": "210371ffeb8f018564e1d07d935dee7f310316c67359ac40569cc74f3c3a5e16"
+        }
+      ],
+      "post-install": [
+        "mkdir -p /app/share/icons/hicolor/512x512/apps",
+        "mkdir -p /app/share/icons/hicolor/64x64/apps",
+        "cp /app/share/nootka/picts/nootka.png /app/share/icons/hicolor/64x64/apps",
```
Comment (nedrichards) [inline net.sf.nootka.json]: I haven't checked yet (I will as it builds) but is this also putting a 128x128px icon in the correct path? It needs that for Software stores.

File: nootka.appdata.xml (hunk 1 above)
Comment (SeeLook) [reply nootka.appdata.xml]: Agree :), fixed

File: nootka.appdata.xml (hunk 2 above)
Comment (SeeLook) [reply nootka.appdata.xml]: Added <releases> tag

File: net.sf.nootka.json (hunk 3 above)
Comment (SeeLook) [reply net.sf.nootka.json]: Ok, done

File: net.sf.nootka.json (hunk 4 above)
Comment (SeeLook) [reply net.sf.nootka.json]: Changed to 'home'

File: net.sf.nootka.json (hunk 5 above)
Comment (SeeLook) [reply net.sf.nootka.json]: Added 128x128 icon. In next stable release I will try to make those icons 'native'. So far 'share/pixmaps' was used for that.

**Comment by SeeLook** [comment]
And thanks a lot for reviewing. I hope it may work fine now.

**Comment by nedrichards** [comment]
bot, build net.sf.nootka

**Comment by nedrichards** [comment]
Repository has been created: https://github.com/flathub/net.sf.nootka

**Pull request closed without merging**